### PR TITLE
- Improve readme and add getting started vignette (#44, @sharlagelfand).

### LINKED ISSUE #42: Improve README, add "Get Started" article
Most of the contents of https://github.com/jennybc/here_here should be part of the README and/or the "Get started" article. The drawing by @allisonhorst could also be added/referenced?

CC @jennybc

### PR DESCRIPTION
**Pull request by sharlagelfand**
Improve README by adding:

* Information on usage (from (https://github.com/jennybc/here_here)[Ode to the here package]).
* Badges (CRAN and lifecycle)
* Allison Horst here illustration

Also added a "Getting Started" vignette which contains most of the same information as the README, but also further details on using here (and why we want to) in an Rmd in a subdirectory.

I assume you'll want to re-render the README so that it doesn't contain the file path to my tidy dev day folder. I'm a little fuzzy on where things (especially the pkgdown docs) might be off since @krlmlr has been making changes to the package while i've been working on it, but happy to work through it. 

Closes #42

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -18,4 +18,8 @@ Date: 2018-02-01
 URL: https://github.com/r-lib/here, http://r-lib.github.io/here
 BugReports: https://github.com/r-lib/here/issues
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.0.2
+RoxygenNote: 6.0.1.9000
```
Comment (krlmlr) [inline DESCRIPTION]: This looks like an artifact from conflict resolution:

```suggestion
RoxygenNote: 7.0.2
```

File: README.md [hunk 2]
```diff
@@ -1,37 +1,65 @@
...
+``` r
+write.csv(iris, here("files", "data", "iris.csv"))
+```
+
+These relative paths work regardless of where the associated source file
+lives inside your project, like analysis projects with data and reports
+in different
+subdirectories.
+
+![](/Users/sharla/github/tidy-dev-day-2020/here/man/figures/illustration.png)<!-- -->
```
Comment (krlmlr) [inline README.md]: Will this work on my machine?

Comment (sharlagelfand) [reply README.md]: 🙃 of course it won't! i thought i was being clever to use `here()` here - relative reference would be much better

**Comment by sharlagelfand** [comment]
thanks, fixed the illustration size + ref!

**Comment by krlmlr** [comment]
Thanks! The docs are in much better shape now!

**Pull request merged**